### PR TITLE
Trusted boot efi

### PR DESCRIPTION
**Pull request by jreidinger**

### CODE REVIEW

File: src/lib/bootloader/grub2efi.rb [hunk 1]
```diff
@@ -109,7 +113,8 @@ def packages
 
     # overwrite BootloaderBase version to save secure boot
```
Comment (ancorgs) [inline src/lib/bootloader/grub2efi.rb]: NP: slightly outdated comment?

File: test/grub2base_test.rb [hunk 2]
```diff
@@ -27,6 +27,15 @@
...
       subject.read
     end
+
+    it "reads trusted boot configuration from sysconfig" do
+      mocked_sysconfig = ::Bootloader::Sysconfig.new(trusted_boot: true)
+      expect(::Bootloader::Sysconfig).to receive(:from_system).and_return(mocked_sysconfig)
+
+      subject.read
+
+      expect(subject.trusted_boot).to eq true
```
Comment (ancorgs) [inline test/grub2base_test.rb]: NP: strictly speaking, nothing ensures this `true` is coming from `mocked_sysconfig.trusted_boot`. I would like to see a complementary case with `false` to be sure.

File: test/grub_install_test.rb [hunk 3]
```diff
@@ -93,6 +93,17 @@ def expect_grub2_install(target, device: nil, removable: false)
 
         subject.execute
       end
+
+      it "pass suse-enable-tpm option when trusted boot is requested" do
```
Comment (ancorgs) [inline test/grub_install_test.rb]: NP: passes

File: src/lib/bootloader/grub2efi.rb (hunk 1 above)
Comment (jreidinger) [inline src/lib/bootloader/grub2efi.rb]: in fact no, as difference between grub2 and grub2efi version is only secure boot, now when both have trusted boot option.

File: test/grub2base_test.rb (hunk 2 above)
Comment (jreidinger) [inline test/grub2base_test.rb]: well, default are false. But I will add it.

**Pull request merged**